lib/ui/recommendation-row.js

```diff
@@ -256,8 +256,10 @@ RecommendationRow.prototype = {
     const val = key.split('.').reduce(function(o, x) {
       return (typeof o == 'undefined' || o === null) ? o : o[x];
     }, this._data);
-    const isEmptyObject = (typeof val == 'object' && !Object.keys(val).length);
-    return (typeof val == 'undefined' || !val || isEmptyObject) ? null : val;
+    let isEmptyObject = val => {
+      return val && typeof val == 'object' && !Object.keys(val).length;
```

Comment:
Ah. If you look at the old line 259, you were doing the Object.keys(val) call without checking that val was truthy. `typeof null` is object, so that check didn't guard against null values.